@Awinja-Andela If you look through the logs from the build that failed, you will notice some errors that say
```
AssertionError: Response didn't redirect as expected: Response code was 200 (expected 302)
```
That is clearly something that is as a result of an assertion statement that does not pass. Something you can fix.

--- a/hc/accounts/tests/test_login.py
+++ b/hc/accounts/tests/test_login.py
@@ -11,6 +11,7 @@ class LoginTestCase(TestCase):
     def test_it_sends_link(self):
         check = Check()
         check.save()
+        count_before = User.objects.count()
 
         session = self.client.session
         session["welcome_code"] = str(check.code)
@@ -23,15 +24,18 @@ class LoginTestCase(TestCase):
 
         
         ### Assert that a user was created
+
         user = User.objects.get(email=form["email"])
-        self.assertTrue(user)
+        count_after = User.objects.count()
+        self.assertEqual(count_after,count_before + 1)
+
 
         # And email sent
         self.assertEqual(len(mail.outbox), 1)
         subject = "Log in to %s" % settings.SITE_NAME
 
         ### Assert contents of the email body
-        self.assertIn('To log into HealthCheck',mail.outbox[0].body)
+        self.assertIn('To log into Healthchecks',mail.outbox[0].body)
 
         ### And check should be associated with the new user
         self.assertEqual(check.get_status("alice@example.org"), "new")

--- a/hc/accounts/tests/test_login.py
+++ b/hc/accounts/tests/test_login.py
@@ -19,7 +19,6 @@ class LoginTestCase(TestCase):
 
         form = {"email": "joan.awinja@andela.com"}
         resp = self.client.post("/accounts/login/", form)
-        # self.assertEqual(resp.status_code, 302)
         self.assertRedirects(resp, "/accounts/login_link_sent/")
 
         
@@ -35,7 +34,7 @@ class LoginTestCase(TestCase):
         subject = "Log in to %s" % settings.SITE_NAME
 
         ### Assert contents of the email body
-        self.assertIn('To log into Healthchecks',mail.outbox[0].body)
+        self.assertIn('To log into {}'.format(settings.SITE_NAME), mail.outbox[0].body)
 
         ### And check should be associated with the new user
         self.assertEqual(check.get_status("alice@example.org"), "new")
@@ -57,7 +56,6 @@ class LoginTestCase(TestCase):
         form = {"email": "alice@example.org"}
 
         r = self.client.post("/accounts/login/", form)
-        # self.assertEqual(r.status_code, 302)
         self.assertRedirects(r, "/accounts/login_link_sent/")
 
 

--- a/hc/accounts/tests/test_profile.py
+++ b/hc/accounts/tests/test_profile.py
@@ -12,39 +12,41 @@ class ProfileTestCase(BaseTestCase):
         self.client.login(username="alice@example.org", password="password")
 
         form = {"set_password": "1"}
-        r = self.client.post("/accounts/profile/", form)
-        assert r.status_code == 302
+        response = self.client.post("/accounts/profile/", form)
+        assert response.status_code == 302
 
         # profile.token should be set now
         self.alice.profile.refresh_from_db()
+
         token = self.alice.profile.token
+
         self.assertTrue(len(token) > 10)
 
-        ### Assert that the token is set
+        # Assert that the token is set
         self.assertTrue(token)
+        self.assertEqual(response.status_code, 302)
 
-        ### Assert that the email was sent and check email content
+        # Assert that the email was sent and check email content
         self.assertEqual(len(mail.outbox), 1)
-        self.assertIn("Here's a link to set a password for your account on My Monitoring Project:", mail.outbox[0].body)
-
-
+        self.assertIn("Here's a link to set a password for your account on Healthchecks", mail.outbox[0].body)
 
     def test_it_creates_api_key(self):
         self.client.login(username="alice@example.org", password="password")
 
         form = {"create_api_key": "1"}
         response = self.client.post("/accounts/profile/", form)
-        
-        ### Test it creates API key
-        self.assertTrue(response)
+
+        # Test it creates API key
+        self.assertContains(response, 'The API key has been created!')
 
     def test_it_revokes_api_key(self):
         self.client.login(username="alice@example.org", password="password")
 
         form = {"revoke_api_key": "1"}
+
         response = self.client.post("/accounts/profile/", form)
-        ### Test it revokes API key
-        self.assertTrue(response)
+        # Test it revokes API key
+        self.assertContains(response, "The API key has been revoked!")
 
     def test_it_sends_report(self):
         check = Check(name="Test Check", user=self.alice)
@@ -52,10 +54,9 @@ class ProfileTestCase(BaseTestCase):
 
         self.alice.profile.send_report()
 
-        ###Assert that the email was sent and check email content
+        # Assert that the email was sent and check email content
         self.assertEqual(len(mail.outbox), 1)
-        self.assertIn("This is a monthly report sent by My Monitoring Project.", mail.outbox[0].body)
-
+        self.assertIn("This is a monthly report sent by Healthchecks.", mail.outbox[0].body)
 
     def test_it_adds_team_member(self):
         self.client.login(username="alice@example.org", password="password")
@@ -68,13 +69,12 @@ class ProfileTestCase(BaseTestCase):
         for member in self.alice.profile.member_set.all():
             member_emails.add(member.user.email)
 
-        ### Assert the existence of the member emails
+        # Assert the existence of the member emails
         self.assertTrue("frank@example.org" in member_emails)
 
-        ###Assert that the email was sent and check email content
+        # Assert that the email was sent and check email content
         self.assertEqual(len(mail.outbox), 1)
         self.assertIn("You will be able to manage their existing monitoring checks and set up new\nones.", mail.outbox[0].body)
-
 
     def test_add_team_member_checks_team_access_allowed_flag(self):
         self.client.login(username="charlie@example.org", password="password")

--- a/hc/accounts/tests/test_profile.py
+++ b/hc/accounts/tests/test_profile.py
@@ -23,12 +23,11 @@ class ProfileTestCase(BaseTestCase):
         self.assertTrue(len(token) > 10)
 
         # Assert that the token is set
-        self.assertTrue(token)
         self.assertEqual(response.status_code, 302)
 
         # Assert that the email was sent and check email content
         self.assertEqual(len(mail.outbox), 1)
-        self.assertIn("Here's a link to set a password for your account on Healthchecks", mail.outbox[0].body)
+        self.assertIn("Here's a link to set a password for your account on {}:".format(settings.SITE_NAME), mail.outbox[0].body)
 
     def test_it_creates_api_key(self):
         self.client.login(username="alice@example.org", password="password")
@@ -56,7 +55,7 @@ class ProfileTestCase(BaseTestCase):
 
         # Assert that the email was sent and check email content
         self.assertEqual(len(mail.outbox), 1)
-        self.assertIn("This is a monthly report sent by Healthchecks.", mail.outbox[0].body)
+        self.assertIn("This is a monthly report sent by {}.".format(settings.SITE_NAME), mail.outbox[0].body)
 
     def test_it_adds_team_member(self):
         self.client.login(username="alice@example.org", password="password")

--- a/hc/settings.py
+++ b/hc/settings.py
@@ -25,8 +25,8 @@ SECRET_KEY = "---"
 DEBUG = True
 ALLOWED_HOSTS = ["*"]
 DEFAULT_FROM_EMAIL = 'healthchecks@example.org'
-USE_PAYMENTS = False
-REGISTRATION_OPEN = False
+USE_PAYMENTS = True
+REGISTRATION_OPEN = True
 
 
 INSTALLED_APPS = (

